atom/browser/notifications/win/win32_desktop_notifications/desktop_notification_controller.cc

```diff
@@ -78,7 +87,7 @@ DesktopNotificationController::DesktopNotificationController(
 
 DesktopNotificationController::~DesktopNotificationController() {
   for (auto&& inst : instances_)
-    DestroyToast(inst);
+    DestroyToast(&inst);
```

Comment:
👍

Reply:
the linter was complaining about non-const references, had to be changed to pointer